test/data/brave_ads/component/rich_media_with_csp_violation/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@aseren lets remove `manifest.json` (this file) because it is not needed for tests.

Reply:
Fixed